src/pages/Assets/Assets.tsx

```diff
@@ -1,14 +1,17 @@
 import { useTranslate } from 'react-polyglot'
+import { useSelector } from 'react-redux'
 import { AssetSearch } from 'components/AssetSearch/AssetSearch'
 import { Main } from 'components/Layout/Main'
 import { SEO } from 'components/Layout/Seo'
+import { selectAssetsByMarketCap } from 'state/slices/selectors'
 
 export const Assets = () => {
   const translate = useTranslate()
+  const assetsByMarketCap = useSelector(selectAssetsByMarketCap)
   return (
     <Main display='flex' flexDir='column' height='calc(100vh - 72px)'>
       <SEO title={translate('navBar.assets')} />
-      <AssetSearch />
+      <AssetSearch assets={assetsByMarketCap} />
```

Comment:
update to use sortedAssets

Reply:
@gomesalexandre im thinking i should make a wrapper component that always injects the assets for us, so we dont forget to input the correct asset selection. This would be used in all cases except tradeinput - thoughts?